source/win-main/App.vue

```diff
@@ -41,8 +41,30 @@
         >
           <template #view1>
             <!-- First side: Editor -->
+            <!--
+              NOTE: The following contains a somewhat counterintuitive logic.
+              What we are effectively doing here is we overlay an editor pane
+              that directly gets passed the node configuration for the editor
+              pane that is currently in distraction free mode (if applicable).
+              The other logic below executes in every case, but if a distraction
+              free pane is shown, they only hide. The benefit is that, while the
+              distraction free pane effectively duplicates the pane and thus has
+              a loading time (especially for larger files), this retains
+              whatever state the other panes and branches were in so that the
+              user can - upon leaving distraction free - immediately continue
+              with their work.
+            -->
+            <EditorPane
```

Comment:
In distraction free mode, there seems to be a grey bar appearing on top of the editor:
![zettlr-20230422](https://user-images.githubusercontent.com/18646029/233784969-4f1837ab-5af4-4485-bad7-e3fd74e2d858.png)
I did some inspecting, and that bar looks like it's part of this EditorPane, but I couldn't find anything more unfortunately. Also, for me, there is a second vertical scrollbar visible on the right.

File I used: "Pandoc and LaTeX Guide" from the Tutorial.

Does this bar also appear on your machine?

Reply:
Found the issue, there was still a `top: 30px` in the Linux styles.